deepchem/transformers/__init__.py

```diff
@@ -95,7 +100,7 @@ def _transform_row(i, df, transformer, data_dir):
 class NormalizationTransformer(Transformer):
 
   def __init__(self, transform_X=False, transform_y=False, transform_w=False,
-               dataset=None):
+               dataset=None, transform_gradients=False):
```

Comment:
@joegomes I think the general question is whether this is a general transformation that we expect to be useful elsewhere. I do think we expect to have many models that work with gradients, but what's the right abstraction to work with these models? Let's chat IRL about these design questions. I think it's harmless to leave this code in here for now.